kvm/v13.1.0/release.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v13.1.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.3.0
+  - name: chart-operator
+    version: 2.5.1
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.3.0
+  - componentVersion: 0.3.6
+    name: metrics-server
+    version: 1.1.1
+  - name: net-exporter
+    version: 1.9.2
+  - componentVersion: 1.0.1
+    name: node-exporter
+    version: 1.7.0
+  components:
+  - name: app-operator
+    version: 2.7.0
+    releaseOperatorDeploy: true
+  - name: calico
+    version: 3.15.3
+  - name: cert-operator
+    releaseOperatorDeploy: true
+    version: 0.1.0
+    reference: 0.1.0-2
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 0.23.19
+  - name: containerlinux
+    version: 2605.10.0
+  - name: etcd
+    version: 3.4.13
+  - name: flannel-operator
+    version: 0.2.0
+  - name: kubernetes
+    version: 1.18.14
```

Comment:
13.0.0 was `1.18.12` -- if this bump is intentional then it should have notes as well

Reply:
Good call. I'll add it.